app/models/manageiq/providers/openstack/inventory/parser/network_manager.rb

```diff
@@ -29,7 +29,9 @@ def cloud_networks
       network.vlan_transparent = n.attributes["vlan_transparent"]
       network.maximum_transmission_unit = n.attributes["mtu"]
       network.cloud_tenant = persister.cloud_tenants.lazy_find(n.tenant_id)
-      network.orchestration_stack = persister.orchestration_stacks.lazy_find(collector.orchestration_stack_by_resource_id(n.id))
+      network.orchestration_stack = persister.orchestration_stacks_resources.lazy_find(
+        collector.orchestration_stack_by_resource_id(n.id).try(:physical_resource_id), :key => :stack
```

Comment:
@mansam so not sure if this is anywhere else, but usually each entity relates to resource, so we wan't lazy_find on resource. But since we want to store stack, we lazily access the stack of the resource by ```:key => :stack```